desc/equilibrium/configuration.py

```diff
@@ -1134,9 +1138,11 @@ def compute(
             data1d = compute_fun(
                 dep1d,
                 params=params,
-                transforms=get_transforms(dep1d, eq=self, grid=grid1d, **kwargs),
-                profiles=get_profiles(dep1d, eq=self, grid=grid1d),
-                data=None,
+                transforms=get_transforms(
+                    dep1d, grid1d.axis.size, eq=self, grid=grid1d, **kwargs
+                ),
+                profiles=get_profiles(dep1d, grid1d.axis.size, eq=self, grid=grid1d),
+                data=None,  # Todo: pass in data0d?
```

Comment:
wasn't an issue now apparently just commented for future. in any case i see why we don't need this logic inside the compute utils dependency building recursion.

Reply:
this is in case there are 1d quantities which rely on 0D quantities?

Reply:
yea i can update the comment to say pass it in when there is one of those